system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/microservices/K8sObjects.js

```diff
@@ -16,18 +16,133 @@
+                                <TableCell className={classes.cellWidth20}>Age</TableCell>
+                                <TableCell className={classes.cellWidth10}></TableCell>
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            <TableRow key="">
+                                <TableCell className={classes.tableCell} component="th" scope="row"></TableCell>
+                                <TableCell className={classes.tableCell}></TableCell>
+                                <TableCell className={classes.tableCell}></TableCell>
+                                <TableCell className={classes.tableCell}></TableCell>
+                                <TableCell className={classes.tableCell}></TableCell>
+                            </TableRow>
+                        </TableBody>
+                    </Table>
+
+                    <Typography color="inherit" className={classes.subtitle}>
+                        Workload
+                    </Typography>
+                    <Table className={classes.table}>
+                        <TableHead>
+                            <TableRow>
+                                <TableCell>Workload</TableCell>
+                                <TableCell className={classes.cellWidth20}>Creation Time</TableCell>
+                                <TableCell className={classes.cellWidth20}>Age</TableCell>
+                                <TableCell className={classes.cellWidth10}></TableCell>
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            <TableRow key="">
+                                <TableCell className={classes.tableCell} component="th" scope="row"></TableCell>
+                                <TableCell className={classes.tableCell}></TableCell>
+                                <TableCell className={classes.tableCell}></TableCell>
+                                <TableCell className={classes.tableCell}></TableCell>
+                            </TableRow>
+                        </TableBody>
+                    </Table>
+                    <Typography color="inherit" className={classes.subtitle}>
+                        Pods
+                    </Typography>
+                    <Table className={classes.table}>
+                        <TableHead>
+                            <TableRow>
+                                <TableCell>Pod</TableCell>
+                                <TableCell>Restarts</TableCell>
+                                <TableCell className={classes.cellWidth20}>Age</TableCell>
+                                <TableCell>Metrics</TableCell>
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            <TableRow key="">
```

Comment:
keys are not necessary

Reply:
removed in https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/common/DataTable.js